MetricsReloaded/metrics/calibration_measures.py

```
@@ -81,15 +100,15 @@ def __init__(
     def class_wise_expectation_calibration_error(self):
-        r"""
+        """
         Class_wise version of the expectation calibration error
 
         Ananya Kumar, Percy S Liang, and Tengyu Ma. 2019. Verified uncertainty calibration. Advances in Neural Information
         Processing Systems 32 (2019).
 
         .. math::
 
-            cwECE = \dfrac{1}{K}\sum_{k=1}^{K}\sum_{i=1}^{N}\dfrac{\vert B_{i,k} \vert}{N} \left(y_{k}(B_{i,k}) - p_{k}(B_{i,k})\right)
+            cwECE = \\dfrac{1}{K}\sum_{k=1}^{K}\sum_{i=1}^{N}\\dfrac{\\vert B_{i,k} \\vert}{N} \\left(y_{k}(B_{i,k}) - p_{k}(B_{i,k})\\right)
```

Ruff (0.15.9)

[warning] Invalid escape sequence: `\s`

Add backslash to escape sequence

(W605)